course/models.py

```diff
@@ -450,6 +450,11 @@ def __str__(self):
     def clean(self):
         super().clean()
         errors = {}
+        RESERVED = ("instances",)
+        if self.instance_name in RESERVED:
+            errors['instance_name'] = _("You cannot use word '{}' as an instance name.").format(self.instance_name)
+        if self.url in RESERVED:
+            errors['url'] = _("You cannot use word '{}' in the url.").format(RESERVED)
```

Comment:
This line has the same problem that was fixed above: `.format(self.instance_name)`

Reply:
Thanks again. I did not see that one